File: Golang/1011.go
package main

import (
	"fmt"
	"math"
)

func main() {
	//volume é: (4/3) * pi * R3
	var raio float64

	fmt.Scan(&raio)

	pi := 3.14159
	volume := (4.0 / 3.0) * pi * math.Pow(raio, 3)
	fmt.Printf("VOLUME = %.3f\n", volume)
}

File: Elixir/1011.exs
raio = IO.gets("") |> String.trim() |> String.to_integer
volume = (4.0 / 3.0) * 3.14159 * (:math.pow(raio, 3))

IO.puts "VOLUME = #{:erlang.float_to_binary(volume, decimals: 3)}"
